extensions/data-transfer/portability-data-transfer-microsoft/src/main/java/org/dataportabilityproject/transfer/microsoft/MicrosoftTransferExtension.java

```diff
@@ -11,25 +11,35 @@
 import org.dataportabilityproject.transfer.microsoft.calendar.MicrosoftCalendarImporter;
 import org.dataportabilityproject.transfer.microsoft.contacts.MicrosoftContactsExporter;
 import org.dataportabilityproject.transfer.microsoft.contacts.MicrosoftContactsImporter;
+import org.dataportabilityproject.transfer.microsoft.derived.MicrosoftDerivedDataExporter;
 import org.dataportabilityproject.transfer.microsoft.photos.MicrosoftPhotosExporter;
 import org.dataportabilityproject.transfer.microsoft.photos.MicrosoftPhotosImporter;
 import org.dataportabilityproject.transfer.microsoft.transformer.TransformerService;
 import org.dataportabilityproject.transfer.microsoft.transformer.TransformerServiceImpl;
 
+/**
+ * Bootstraps the Microsoft data transfer services. Note the format of the exported contents are
+ * opaque; they may change without notice.
```

Comment:
is this comment true of all data or just derived?

Reply:
Good catch, that's not true. I moved it out to the derived data exporter.